.github/workflows/pr-verify.yml

```
@@ -175,7 +176,7 @@ jobs:
         uses: actions/configure-pages@v3
       - name: Build Docs
         run: |
-          RUSTDOCFLAGS="--enable-index-page -Zunstable-options" cargo doc --no-deps --all-features
+          RUSTDOCFLAGS="--enable-index-page -Zunstable-options" cargo +nightly doc --no-deps --all-features
```

Just found out about `RUSTC_BOOTSTRAP=1`, it can be used to turn your stable rustc into a nightly one. Recently used it to take `nightly` dependence out of code coverage. 

```suggestion
          RUSTC_BOOTSTRAP=1 RUSTDOCFLAGS="--enable-index-page -Zunstable-options" cargo  doc --no-deps --all-features
```

---

thanks, will apply. Does this only work for docs or we can replace +nightly for other things like fomatting?